### PR TITLE
Beta fixes 2

### PR DESCRIPTION
**Pull request by jonzee**
#79 
#82
#85  
#89 
#90

### CODE REVIEW

File: src/ts/Page/pages.ts [hunk 1]
```diff
@@ -85,7 +86,7 @@ export default (() => {
             body: 'Our team will contact you within 24 hours and work together to create an offer of your motorbike.',
           },
           image1: 'https://res.cloudinary.com/hsv4d1yfs/image/upload/v1497815392/page/owner1.jpg',
-          image2: 'https://res.cloudinary.com/hsv4d1yfs/image/upload/v1497815392/page/owner2.jpg',
+          imageb: 'https://res.cloudinary.com/hsv4d1yfs/image/upload/v1497815392/page/owner2.jpg',
```
Comment (michalmikolajczyk) [inline src/ts/Page/pages.ts]: wtf?

File: src/ts/app.tsx [hunk 2]
```diff
@@ -1,3 +1,6 @@
+import Promise from 'promise-polyfill';
+if (!window['Promise']) window['Promise'] = Promise;
```
Comment (michalmikolajczyk) [inline src/ts/app.tsx]: we could handle this with `modernizr` I guess

File: src/ts/API/index.tsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
 const url = (() => {
-  if (typeof location === 'undefined') return 'https://viking-garage-api-prod.herokuapp.com';
+  if (typeof location === 'undefined') return 'https://viking-garage-api-production.herokuapp.com';
   if (location.hostname.match('localhost')) return 'https://viking-garage-api-dev.herokuapp.com';
   if (location.hostname.match('dev')) return 'https://viking-garage-api-dev.herokuapp.com';
   if (location.hostname.match('staging')) return 'https://viking-garage-api-staging.herokuapp.com';
+  if (location.hostname.match('vikinggarage')) return 'https://viking-garage-api-production.herokuapp.com';
```
Comment (michalmikolajczyk) [inline src/ts/API/index.tsx]: let's be more precise here, the actual hostname is `vikinggarage.com`, only that one should match production API

File: src/ts/API/index.tsx [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
   if (location.hostname.match('viking-garage-frontend-s-pr')) return 'https://viking-garage-api-staging.herokuapp.com';
-  return 'https://viking-garage-api-dev.herokuapp.com';
+  return 'https://viking-garage-api-production.herokuapp.com';
```
Comment (michalmikolajczyk) [inline src/ts/API/index.tsx]: it should point to staging at that point

File: src/ts/API/index.tsx [hunk 5]
```diff
@@ -1,10 +1,11 @@
 const url = (() => {
-  if (typeof location === 'undefined') return 'https://viking-garage-api-prod.herokuapp.com';
+  if (typeof location === 'undefined') return 'https://viking-garage-api-production.herokuapp.com';
```
Comment (michalmikolajczyk) [inline src/ts/API/index.tsx]: if `unknown` then point to staging API

File: src/ts/app.tsx (hunk 2 above)
Comment (jonzee) [inline src/ts/app.tsx]: Ok, I'll add `modernizr` in next `PR` `beta-fixes-3`

**Pull request merged**